src/add/operator/multicast.ts

```diff
@@ -2,7 +2,7 @@
 import {Observable} from '../../Observable';
 import {multicast, MulticastSignature} from '../../operator/multicast';
 
-Observable.prototype.multicast = multicast;
+Observable.prototype.multicast = <any>multicast;
```

Comment:
To support different signature `multicast(subject)` / `multicast(factory, selector?)`.

As similar to other operators have different signatures [`mergeMapTo`](https://github.com/ReactiveX/rxjs/blob/7cd8a4365c369a2d6e49ffa1dde5659057f6aad8/src/operator/mergeMapTo.ts#L63-L67) / [`add/mergeMapTo`](https://github.com/ReactiveX/rxjs/blob/7cd8a4365c369a2d6e49ffa1dde5659057f6aad8/src/add/operator/mergeMapTo.ts#L6]) if there's completely different two signatures, can't be assigned into prototype directly since function definition is single, polymorphic.